boards/shields/x_nucleo_nfc04a1/doc/index.rst

```diff
@@ -0,0 +1,69 @@
+.. _x-nucleo-nfc04a1:
+
+X-NUCLEO-NFC04A1: MEMS Inertial and Environmental Multi sensor shield
+#####################################################################
+
+Overview
+********
+The X-NUCLEO-NFC04A1 dynamic NFC/RFID tag IC expansion board is based on the
+ST25DV04K NFC Type V/RFID tag IC with a dual interface 4 Kbits EEPROM that
+also features an I²C interface. It can be powered by the pin of Arduino
+connector or directly by the received carrier electromagnetic field.
+
+The X-NUCLEO-NFC04A1 expansion board is compatible with the Arduino
+UNO R3 connector pin assignment and can easily be plugged onto any STM32
+Nucleo board. Various expansion boards can also be stacked to evaluate
+different devices operating together with the dynamic NFC tag. The board
+also features an antenna with a 54 mm ISO 24.2 diameter, single layer,
+copper etched on PCB.
+
+.. image:: img/x-nucleo-nfc04a1.jpg
+     :align: center
+     :alt: X-NUCLEO-NFC04A1
```

Comment:
```suggestion
     :alt: X-NUCLEO-NFC04A1

     X-NUCLEO-NFC04A1 shield (Credit: STMicroelectronics)
```

Reply:
Thanks, I updated the doc to add the credit